src/dotVariant.Generator/templates/Variant.scriban-cs

```diff
@@ -97,20 +97,32 @@ namespace {{ Variant.Namespace }}
         public {{ method_modifiers }}bool Equals({{ variant_nullable_param }} other)
             => {{ if Variant.IsReferenceType; "!(other is null) && "; end }}_variant.Equals(other._variant);
 
+        /// <summary>Returns a value that indicates whether two variants are equal.</summary>
```

Comment:
```suggestion
        /// <summary>Compare two <see cref="{{ Variant.Type }}" /> instances for equality.</summary>
```

Reply:
Yeah, probably more fitting to include the specific type as well.